firmware/src/hal/sgx/test/secret_store/test_secret_store.c

```diff
@@ -40,6 +40,31 @@
 #define MAX_BLOB_SIZE (1024 * 1024)
 // Utility macro that converts a plaintext secret into the sealed version
 #define SEALED(str) ("SEALED - " str)
+// The sha256 hash of the key "key"
+#define KEY_HASH                               \
+    "\x2c\x70\xe1\x2b\x7a\x06\x46\xf9\x22\x79" \
+    "\xf4\x27\xc7\xb3\x8e\x73\x34\xd8\xe5\x38" \
+    "\x9c\xff\x16\x7a\x1d\xc3\x0e\x73\xf8\x26" \
+    "\xb6\x83"
+
+// Used to store a plaintext secret and the expected sealed and
+// unsealed versions
+typedef struct {
+    // The original plaintext secret
+    uint8_t* plaintext;
+    // The size of the plaintext secret
+    size_t plaintext_size;
+    // The unsealed secret, which is the plaintext secret prepended with the
+    // sha256 hash of the key
+    uint8_t* unsealed_secret;
+    // The size of the unsealed secret
+    size_t unsealed_size;
+    // The sealed secret, which is the unsealed secret prepended with the string
+    // "SEALED - "
+    uint8_t* sealed_secret;
+    // The size of the sealed secret
+    size_t sealed_size;
+} secret_t;
```

Comment:
I find the name of this mocking structure very confusing due to the existence of `secret_t` in the secret store implementation. Can we give this a different and more meaningful name please?

Reply:
Sounds good, I'll rename it so that it makes its meaning clearer